package.json

```diff
@@ -37,8 +37,8 @@
     "minimatch": "^3.0.0",
     "progress": "^1.1.8",
     "shelljs": "^0.5.3",
-    "typedoc-default-themes": "0.4.0",
-    "typescript": "1.7.3"
+    "typedoc-default-themes": "sierrasoftworks/typedoc-default-themes#v0.3.5",
```

Comment:
Thanks. Hopefully @sebastian-lenz will come back online long enough to give me access to both of these modules.